src/test/regress/expected/multi_create_table_constraints.out

```diff
@@ -50,6 +60,16 @@ CREATE TABLE uq_on_non_part_col
 SELECT create_distributed_table('uq_on_non_part_col', 'partition_col', 'hash');
 ERROR:  cannot create constraint on "uq_on_non_part_col"
 DETAIL:  Distributed relations cannot have UNIQUE, EXCLUDE, or PRIMARY KEY constraints that do not include the partition column (with an equality operator if EXCLUDE).
+-- check that we can disable the constraint check
+BEGIN;
+SET LOCAL citus.allow_unsafe_constraints TO on;
+SELECT create_distributed_table('uq_on_non_part_col', 'partition_col', 'hash');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
```

Comment:
Should we at least print the index definition on one of the shards and at least on one of the (worker node) shell tables to make sure that we're creating the index correctly ?